packages/@sanity/types/src/schema/definition/schemaDefinition.ts

```diff
@@ -115,6 +114,8 @@ export interface TypeAliasDefinition<
   preview?: PreviewConfig
 
   components?: {
+    annotation?: ComponentType<any>
```

Comment:
I'm looking forward to being able to add TSDocs to everything, when we have ref-doc generation in place. I think it is needed a lot of places like this, to add breadcrumbs to when this is useful.